File: Problems_and_Solutions/days-10 to 19/day-10/cpp/solution 3.cpp
class Solution {
  public:
    // Function to count nodes of a linked list.
    int getCount(struct Node* head) {

        // Code here
        int count=1;
        while(head->next!=NULL)
        {
            count++;
            head=head->next;
        }
        return count;
    }
};

File: Problems_and_Solutions/days-10 to 19/day-10/Java/Solution3.java
class Node {
    int data;
    Node next;
    Node(int data) {
        this.data = data;
        this.next = null;
    }
}

class Solution3 {
    public static int getCount(Node head) {
        int count = 0;
        Node temp = head;
        while(temp != null){
            count++;
            temp = temp.next;
        }
        return count;
    }

    public static void main(String[] args) {
        Node head = new Node(1);
        head.next = new Node(2);
        head.next.next = new Node(3);

        System.out.println(getCount(head));
    }
}